### PR TITLE
v1 parity

### PR DESCRIPTION
**Pull request by chr0n1x**

### CODE REVIEW

File: v2/fleet/datadog.service [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+[Unit]
+Description=Datadog Agent
+After=docker.service
+
+[Service]
+User=core
+Restart=always
+ExecStartPre=-/usr/bin/docker stop dd-agent
+ExecStartPre=-/usr/bin/docker rm -f dd-agent
+ExecStartPre=/usr/bin/docker pull behance/docker-dd-agent:latest
```
Comment (cfortier2) [inline v2/fleet/datadog.service]: can we move the pull to be the first Pre command please.

File: v2/fleet/sumologic.service [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ExecStartPre=-/usr/bin/docker rm -f sumologic
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill sumologic
+ExecStartPre=-/usr/bin/docker rm sumologic
+ExecStart=/usr/bin/bash -c \
+    "if [[ -f /etc/profile.d/etcdctl.sh ]]; then source /etc/profile.d/etcdctl.sh;fi && \
+    sudo /usr/bin/docker run --name sumologic \
+    -v /var/lib/docker/containers/:/tmp/clogs/:ro \
+    -e SUMO_NAME=docker_cluster \
+    -e SUMO_CATEGORY=be/${NODE_PRODUCT}/${NODE_TIER}/control-logs \
```
Comment (iderdik) [inline v2/fleet/sumologic.service]: In the worker tier, this should be `-e SUMO_CATEGORY=be/${NODE_PRODUCT}/${NODE_TIER}/container-logs`

We want to keep the FD/Mesos/Marathon/FD logs in a different sourceContainer than the App logs

File: v2/fleet/logrotate.service [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+[Service]
+User=core
+Restart=always
+ExecStartPre=/usr/bin/docker pull behance/docker-gocron-logrotate
+ExecStartPre=-/usr/bin/docker stop logrotate
+ExecStartPre=-/usr/bin/docker rm -f logrotate
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill logrotate
+ExecStartPre=-/usr/bin/docker rm logrotate
```
Comment (cfortier2) [inline v2/fleet/logrotate.service]: this is here twice.

File: v2/fleet/sumologic-control.service [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+[Service]
+EnvironmentFile=/etc/environment
+User=core
+Restart=always
+ExecStartPre=/usr/bin/docker pull behance/docker-sumologic
+ExecStartPre=-/usr/bin/docker stop sumologic-control
+ExecStartPre=-/usr/bin/docker rm -f sumologic-control
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill sumologic-control
+ExecStartPre=-/usr/bin/docker rm sumologic-control
```
Comment (cfortier2) [inline v2/fleet/sumologic-control.service]: this is here twice.

File: v2/fleet/sumologic.service [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+[Service]
+EnvironmentFile=/etc/environment
+User=core
+Restart=always
+ExecStartPre=/usr/bin/docker pull behance/docker-sumologic
+ExecStartPre=-/usr/bin/docker stop sumologic
+ExecStartPre=-/usr/bin/docker rm -f sumologic
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill sumologic
+ExecStartPre=-/usr/bin/docker rm sumologic
```
Comment (cfortier2) [inline v2/fleet/sumologic.service]: duplicate.

Comment (iderdik) [inline v2/fleet/sumologic.service]: Sumologic and sumologic-control are 2 services to allow us to log 2 things to different source categories. They's don't run on the same tiers.

**Comment by iderdik** [comment]
:+1:

**Pull request merged**